vpr/src/route/rr_graph2.cpp

```diff
@@ -2333,7 +2338,7 @@ static void get_switchblocks_edges(RRGraphBuilder& rr_graph_builder,
                 ++edge_count;
 
                 //we only add the following edge between intermediate nodes once for the first 3D connection for each pair of intermediate nodes
```

Comment:
Is there a big comment somewhere saying how custom_3d_sb_fanin_fanout works?  It should detail exactly what the rr graph this generates looks like.